mm2src/mm2_bin_lib/build.rs

```diff
@@ -51,31 +51,26 @@ fn mm_version() -> String {
     // and a release tag when building from some kind of a stable branch,
     // though we should keep the ability for the tooling to provide the “MM_VERSION”
     // externally, because moving the entire ".git" around is not always practical.
```

Comment:
This comment should be moved/updated and the doc comment above needs updating too (Remove MarketMaker and use API instead, etc..)